timemachine/md/moves.py

```diff
@@ -95,6 +95,20 @@ def move(self, x: _State) -> _State:
         return x
 
 
+class BatchedMixtureOfMoves(MixtureOfMoves[_State]):
+    """Apply a single move uniformly selected from a list"""
```

Comment:
```suggestion
    """Repeatedly sample from a uniform mixture of moves"""
```
or similar. (To clarify that this is not "batching" over `len(mixture)` but over `num_iterations`.)

Reply:
Added in https://github.com/proteneer/timemachine/pull/1260/commits/ae4f61c02a02ee829ee11ae560e381c6f03478c7